pandas/tseries/period.py

```diff
@@ -281,6 +281,16 @@ def __contains__(self, key):
             return False
         return key.ordinal in self._engine
 
+    def __array_wrap__(self, result, context=None):
```

Comment:
doesn't this need to go in ``tseries/base``, e.g. ufuncs are not valid for any datetimelike?